### ISSUE TITLE
Use semantic versioning

### ISSUE THREAD

**Issue by SamMousa** [PHPOffice/PHPExcel#758]
It would be great if you could use semantic versioning!

**Comment by MarkBaker** [comment]
Which element of [semantic versioning](http://semver.org/) are we failing on? It would be great if you'd explain what we're doing wrong, in words that I can understand

**Comment by SamMousa** [comment]
Well the most important one I noticed is increasing the major version number when you break backwards compatibility. Currently between 1.9 is not BC with 1.8 and thus should be numbered 2.x.y.

**Comment by MarkBaker** [comment]
The 1.9 branch isn't in any way stable yet, and shouldn't be used: and unlikely to ever see light of day as PHPExcel 1.9.0..... it's more likely to be 1.0.0 of PHPSpreadsheet instead.... 1.9 is where I'm posting the code changes as I do them so that there's a copy at least if I get hit by a bus, or if the cat fritzes my laptop next time she urinates over it

**Comment by SamMousa** [comment]
@MarkBaker don't get hit by a bus, there will be so much delay for the passengers!

I have my minimum stability set to dev, since I do not mind having buggy code (ie code with possible regressions), which is the case for the 1.8.x-dev branch, but now composer will pick up 1.9.x-dev since it thinks it is a compatible dev release.

Anyway, I've double checked and packagist correctly marks 1.8.1 as the latest stable.
Of course others don't know about the plan to not ever release 1.9.x-dev as 1.9.x ;-) It would be more intuitive to either move it to a different repo, or name it 2.0.x-dev instead, so that we can see you're working on a new major release. 
It would also help possible new contributors (I don't want to contribute to 1.8.x if instead I can help get a new major version out instead for example).

Anyway, given the state of 1.9.x-dev it might not be so urgent!

**Comment by sergeyklay** [comment]
+1

**Comment by MarkBaker** [comment]
Is that a +1 to my not getting hit by a bus?

**Comment by sergeyklay** [comment]
:smile: 
I fully agree with @SamMousa

**Comment by ctrl-f5** [comment]
agree, I just installed using `composer require phpoffice/phpexcel` and was stumped that there was no PHPExcel class present... Major BC breaks call for major version changes.
